conflict_resolution.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>
+        Positive Work Environment at W3C: Conflict Resolution Procedures
+        </title>
+        <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"
+        defer="defer"></script>
+        <script class='remove'>
+        // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+        var respecConfig = {
+        edDraftURI: "https://w3c.github.io/PWETF/",
+        specStatus: "ED",
+        shortName: "pwe-crp",
+        editors: [{
+            name: "Tzviya Siegman",
+            company: "Wiley",
+            companyURL: "https://www.wiley.com",
+            w3cid: 65542
+        }, {
+            name: "Wendy Reid",
+            company: "Rakuten Kobo",
+            companyURL: "https://rakuten.com",
+            w3cid: 102009
+        }],
+        github: {
+            repoURL: "https://github.com/w3c/PWETF/",
+            branch: "main"
+        },
+        };
+        </script>
+    </head>
+    <body>
+        <section id="abstract">
+            <p>This document is a companion to the W3C Code of Conduct which clarifies the procedures for handling incidents relating to Code of Conduct violations or complaints.</p>
+        </section>
+
+        <section id="sotd"></section>
+
+        <section id="introduction">
+            <h2>Introduction</h2>
+
+            <p>As we engage with one another, incidents can happen. Incidents can take many forms, but participants have the right to a safe and positive work environment. The goal of conflict resolution is that participants in W3C can:</p>
```

Comment:
+1 to the word "incident".  Excellent word choice.

Suggest: s/conflict resolution/this incident resolution procedure/